src/main/java/edu/ucsb/nceas/mdqengine/Controller.java

```diff
@@ -613,6 +694,39 @@ public void handleDelivery(String consumerTag, Envelope envelope, AMQP.BasicProp
         RabbitMQchannel.basicConsume(COMPLETED_QUEUE_NAME, false, consumer);
     }
 
+    /**
+     * Monitors for pids stuck processing using Quartz Scheduler.
+     * Starts the scheduler, creates the MonitorJob, and schedules it with a cron
+     * trigger, the schedule of which is stored in metadig.properties as
+     * quartz.monitor.schedule.
+     */
```

Comment:
Helpful to explain the monitor lifecycle here -- mainly the "why" and how the stuck jobs come about, and what happens when they do.